src/arr/compiler/pyret.arr

```diff
@@ -89,15 +93,16 @@ fun main(args):
         B.set-allow-builtin-overrides(r.get-value("allow-builtin-overrides"))
       end
       if not(is-empty(rest)):
-        raise("No longer supported")
+        _ = print("No longer supported\n")
+        failure-code
       else:
         if r.has-key("build-runnable") block:
           outfile = if r.has-key("outfile"):
             r.get-value("outfile")
           else:
             r.get-value("build-runnable") + ".jarr"
           end
-          CLI.build-runnable-standalone(
+          _ = CLI.build-runnable-standalone(
```

Comment:
Fixed in https://github.com/brownplt/pyret-lang/commit/09c4fa85c194b9751fb1a7a97a7f472fdeea11e2